Tests/NumLib/TestComponentNorms.cpp

```diff
@@ -143,7 +143,7 @@ TEST(MPITest_NumLib, ComponentNormMultiComponent1)
     unsigned const num_components = 3;
     auto const norm_type = MathLib::VecNormType::NORM1;
     auto const tolerance =
-        num_components * 600 * std::numeric_limits<double>::epsilon();
```

Comment:
Through the incremental increase this is now at 1700 * eps.

No vectorization also means that intermediate results can be stored (in registers) with 80bit precision, instead of 64 bit when using sse/avx.